README.md

```diff
@@ -59,6 +63,28 @@ render(myAsyncComponent).then(function (html) {
 })
 ```
 
+Sync rendering
+--------------
+
+You can also render synchronously. You just don't get the `waitFor` callback.
+
+```js
+var myAsyncComponent = {
+  oninit: function (node, waitFor) {
+    // waitFor === undefined
+    new Promise(function (resolve) {
+      node.state.foo = 'bar'
+      resolve()
+    })
+  },
+  view: function (node) {
+    return m('div', node.state.foo)
+  }
+}
+
+var html = render.sync(myAsyncComponent)
```

Comment:
does this really work? Don't get how you where able to convert a async call to a sync one.

Reply:
will try that, if I found time at the weekend